py3status/modules/battery_level.py

```diff
@@ -74,18 +81,21 @@
         (default None)
 
 Requires:
-    - the `acpi` command line
+    - the `acpi` the acpi command line utility (only if
+    `measurement_mode='acpi'`)
```

Comment:
don't worry about the README.md changes we can fix them later

this change is the really important line from my point of view
```
    acpi: command line utility (only if
```

> FileNotFOundError.

Do you have a traceback for that?  I think other people have had this issue and it may depend on how you have things installed